librocksdb_sys/crocksdb/c.cc

```diff
@@ -1845,11 +1881,22 @@ struct crocksdb_eventlistener_t : public EventListener {
                                   const crocksdb_compactionjobinfo_t*);
   void (*on_external_file_ingested)(
       void*, crocksdb_t*, const crocksdb_externalfileingestioninfo_t*);
+  void (*on_stall_conditions_changed)(void*, const crocksdb_writestallinfo_t*);
 
   virtual void OnFlushCompleted(DB* db, const FlushJobInfo& info) {
     crocksdb_t c_db = {db};
     on_flush_completed(state_, &c_db,
                        reinterpret_cast<const crocksdb_flushjobinfo_t*>(&info));
+
+    {
+      #ifdef TEST_STALL_CONDITIONS_CHANGED
```

Comment:
Seems very hacky, RocksDB provides a `PauseBackgroundWork()` function, maybe you can try to use it and flush some memtables to see if you can trigger a write stall.